packages/e2e/src/rpc-spec.test.ts

```diff
@@ -0,0 +1,65 @@
+import { ApiPromise } from '@polkadot/api'
+import { describe, expect, it } from 'vitest'
+import { dev, env, observe, setupPolkadotApi, testingPairs } from './helper.js'
+
+const testApi = await setupPolkadotApi(env.acalaV15)
+
+const { alice, bob } = testingPairs()
+
+describe('transaction_v1', async () => {
+  it('sends and executes transactions', async () => {
+    const chainHead = testApi.observableClient.chainHead$()
+
+    const api = await prepareChainForTx()
+
+    const tx = await api.tx.balances.transferKeepAlive(bob.address, 100n).signAsync(alice)
+    const { nextValue, subscription } = observe(chainHead.trackTx$(tx.toHex()))
+    const resultPromise = nextValue()
+    const broadcast = testApi.observableClient.broadcastTx$(tx.toHex()).subscribe()
+
+    // We don't have a confirmation of when the transaction has been broadcasted through the network
+    // it just continues to get broadcasted through the nodes until we unsubscribe from it.
+    // In this case, where there's only one node, a couple of blocks should be enough.
+    await dev.newBlock()
+    const hash = await dev.newBlock()
```

Comment:
tx is included immediately, why do we need 2 blocks?

Reply:
polkadot-api synchronously sends the broadcast, but that goes through the websocket connection, which is async. Unfortunately we don't have a confirmation when the broadcast is completed (since the node just keeps broadcasting it to every node).

Realistically, I could wait a fixed amount of time (I saw ~250ms is enough to have the transaction included in the tx pool), but waiting for two blocks felt a bit closer to what happens in a real chain too.